src/runtime/opencl.cpp

```diff
@@ -99,6 +103,15 @@ WEAK char build_options[1024];
 WEAK ScopedSpinLock::AtomicFlag build_options_lock = 0;
 WEAK bool build_options_initialized = false;
 
+template<typename... Args>
+halide_error_code_t error_opencl(void *user_context, cl_int cl_error, const Args &...args) {
+    if (cl_error == CL_SUCCESS) {
+        return halide_error_code_success;
+    }
+    error(user_context).append(get_opencl_error_name(cl_error), args...);
```

Comment:
There might need to be a " " between the opencl error name and the rest of the args. As far as I understand, .append doesn't add spaces. Comment also applies to other gpu runtimes.

Reply:
Also see above comment I just added to cuda.cpp

Reply:
> There might need to be a " " between the opencl error name and the rest of the args. As far as I understand, .append doesn't add spaces. Comment also applies to other gpu runtimes.

Yeah, I just now removed the 'sep' argument entirely